pkg/gcc/send_side_bwe.go

```diff
@@ -131,6 +135,10 @@ func (e *SendSideBWE) AddStream(info *interceptor.StreamInfo, writer interceptor
 
 // WriteRTCP adds some RTCP feedback to the bandwidth estimator
 func (e *SendSideBWE) WriteRTCP(pkts []rtcp.Packet, attributes interceptor.Attributes) error {
+	if e.IsClosed() {
```

Comment:
I think we still need the lock to make sure the interceptor doesn't get closed while looping over `pkts`. Maybe we should use a different lock that only locks access to `e.close` and does not block all the other places that currently use `e.lock`.

Reply:
Done